src/parser/shared/normalizers/MissingCasts.js

```diff
@@ -0,0 +1,42 @@
+import EventsNormalizer from 'parser/core/EventsNormalizer';
+import SPELLS from 'common/SPELLS';
+
+/**
+ * During analysis there's no way to know at a `begincast` event if it will end up being canceled. This marks all `begincast` events by the player with an `isCancelled` property whether it was cancelled.
```

Comment:
this description seems invalid

Reply:
dammit I keep forgetting to change comments when I copy code from another module, fixing that now